### ISSUE TITLE
Consistent division by zero semantic

### ISSUE THREAD

**Issue by oscbyspro** [oscbyspro/AwesomeNumbersKit#132]
When dividing by zero, Swift has the semantics of returning the truncated (if needed) dividend. Currently, I'm returning the divisor for single digit remainder operations. This is because the divisor always fits in the remainder. At some point, I changed this in [Numberick](https://github.com/oscbyspro/Numberick) when I concluded that having greater consistency would be better, and I would like to port this change. These values are by definition junk values - but I still believe that consistent junk is preferred over inconsistent junk.

> Too long; didn't read: division by zero will return `Output(truncatingIfNeeded: dividend)` similar to Swift's stdlib.

**Comment by oscbyspro** [comment]
TLDR; division by zero will return `Output(truncatingIfNeeded: dividend)`.
